src/app/services/itinerary.service.ts

```diff
@@ -10,12 +11,14 @@ export class ItineraryService {
 
   constructor(private httpClient: HttpClient) { }
 
-  hostURL = 'http://localhost:3000/';
+  hostURL = 'https://be-dev-yourney.herokuapp.com/';
 
-  public lista(): Observable<Itinerary[]> {
-    var url=this.hostURL + 'itineraries'
-    var req=this.httpClient.get<Itinerary[]>(url);
-    req.subscribe(res=>console.log(res))
+  public userItineraries(page:Number, userId:Number): Observable<ItineraryUserPage> {
+    var url=this.hostURL + 'itinerary/user/'+page;
+    //const headers = { 'Authorization': 'Bearer my-token', 'My-Custom-Header': 'foobar' };
+    var body_params = { "userId": userId };
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
    var body_params = { userId };
```